cms/jinja2/templates/pages/information_page.html

```diff
@@ -0,0 +1,34 @@
+{% extends "templates/base_page.html" %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+{% block main %}
+    <p>{{ page.summary }}</p>
+
+    {% if page.last_updated %}
+        <h5>Last Updated: {{ page.last_updated }}</h5>
+    {% endif %}
+
+    {% include_block page.content %}
+
+    {% if related_pages %}
+        {# fmt:off #}
+        {{-
+            onsRelatedContent({
+                "ariaLabel": _('Related content'),
+                "rows": [{
+                    "id": 'related-content',
+                    "title": _('Related content'),
+                    "itemsList": related_pages
+                }]
+            })
+        -}}
+        {# fmt:on #}
+    {% endif %}
+
+{% endblock %}
+{% if page.has_equations %}
+    <script async src="{{ mathjax() }}"></script>
+{% endif %}
+{% if page.has_ons_embed %}
+    <script src="{{ static('js/third-party/pym.min.js') }}"></script>
+{% endif %}
```

Comment:
```suggestion
```

These should not be needed as we include then in `base_page.html` and your page model inherits from `BasePage` which adds the generic `has_equations` / `has_ons_embed`

Reply:
Understood 🫡